src/main/webapp/app/shared/util/core-submission/core-submission.spec.ts

```diff
@@ -0,0 +1,659 @@
+      ['mutations/0', undefined],
+      ['summary', { type: EvidenceEvidenceTypeEnum.GeneSummary, updateTime, gene, drugListRef, entrezGeneId }],
+      ['background', { type: EvidenceEvidenceTypeEnum.GeneBackground, updateTime, gene, drugListRef, entrezGeneId }],
+      [
+        'mutations/0/mutation_effect/oncogenic',
+        { type: EvidenceEvidenceTypeEnum.Oncogenic, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/summary',
+        { type: EvidenceEvidenceTypeEnum.GeneTumorTypeSummary, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/prognosticSummary',
+        { type: EvidenceEvidenceTypeEnum.PrognosticSummary, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/diagnosticSummary',
+        { type: EvidenceEvidenceTypeEnum.DiagnosticSummary, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/mutation_effect/oncogenic',
+        { type: EvidenceEvidenceTypeEnum.Oncogenic, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/mutation_effect/effect',
+        { type: EvidenceEvidenceTypeEnum.MutationEffect, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/prognostic/description',
+        { type: EvidenceEvidenceTypeEnum.PrognosticImplication, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/diagnostic/description',
+        { type: EvidenceEvidenceTypeEnum.DiagnosticImplication, tumor, mutation, updateTime, gene, drugListRef, entrezGeneId },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/0/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.InvestigationalTherapeuticImplicationsDrugSensitivity,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiIs,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/1/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.InvestigationalTherapeuticImplicationsDrugResistance,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiIr,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/2/treatments/0',
+        {
+          type: EvidenceEvidenceTypeEnum.StandardTherapeuticImplicationsForDrugSensitivity,
+          tumor,
+          mutation,
+          updateTime,
+          gene,
+          ti: tiSs,
+          drugListRef,
+          treatment,
+          entrezGeneId,
+        },
+      ],
+      [
+        'mutations/0/tumors/0/TIs/3/treatments/0',
```

Comment:
Does this mean anything under the treatment will trigger this evidence to be generated?

Reply:
everything except short and indication

Reply:
when short changes the evidence should not be populated